sympy/combinatorics/schur_number.py

```diff
@@ -0,0 +1,120 @@
+"""
+The Schur number S(k) is the largest integer n for which the interval [1,n]
+can be partitioned into k sum-free sets.(http://mathworld.wolfram.com/SchurNumber.html)
+"""
+import math
+from sympy import Symbol
+from sympy.core import S
+from sympy.core.function import Function
+
+
+class schur_number_lower_bound(Function):
+    """
+    Gives the value for 1/2 * (3**n  - 1)
+    """
+
+    @classmethod
+    def eval(cls, k):
+        expr = (1/2) * (3**k - 1)
```

Comment:
Don't write 1/2 directly in a SymPy expression. That will cause it to create a float. See https://docs.sympy.org/latest/tutorial/gotchas.html. A better way to write this is `(3**k - 1)/2`.

Reply:
@asmeurer I will update this in the next PR.